toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -702,6 +708,14 @@ void FarmServer::queryHwInfo(HwInfo &hwInfo)
 	len = sizeof(physMemSize);
 	sysctl(mib, 2, &physMemSize, &len, NULL, 0);
 
+#elif defined(LINUX)
+	physMemSize = (TINT64)sysconf(_SC_PHYS_PAGES) * (TINT64)sysconf(_SC_PAGE_SIZE);
```

Comment:
Setting to 500mb is done only for platforms besides Linux/OSX/Win32.
Currently these systems aren't supported anyway. We could do an `#error` instead too.